distributions/distribution-resources/src/main/resources/deb/systemd/openhab2.service

```diff
@@ -1,6 +1,7 @@
 [Unit]
-Description=Starts and stops the openHAB Home Automation Bus
-Documentation=http://www.openhab.org
+Description=openHAB 2 Home Automation Bus Solution
+Documentation=http://docs.openhab.org
```

Comment:
"A space-separated list of URIs... This option may be specified more than once, in which case the specified list of URIs is merged."
Other famous services like samba follow this strategy.